packages/server/utils/OpenAIServerManager.ts

```diff
@@ -20,20 +26,28 @@ class OpenAIServerManager {
     if (!this.openAIApi) return null
     try {
       const location = summaryLocation ?? 'retro meeting'
-      const response = await this.openAIApi.createCompletion({
-        model: 'text-davinci-003',
-        prompt: `Below is a comma-separated list of text from a ${location}. Summarize the text for a second-grade student in one or two sentences.
+      const response = await this.openAIApi.createChatCompletion({
+        model: 'gpt-3.5-turbo',
+        messages: [
+          {
+            role: 'user',
+            content: `Below is a comma-separated list of text from a ${location}.
+            Summarize the text for a second-grade student in one or two sentences.
+            If you can't provide a summary, simply write the word "${OpenAIMagicWords.NO_SUMMARY_PROMPT_INSTRUCTION_WORDS}".
```

Comment:
@tianrunhe when you get the chance, would you mind adding the new gender pronoun line to this prompt: `When referring to people in the summary, do not assume their gender and default to using the pronouns "they" and "them".`?